QUEST.tsv

```diff
@@ -2777,42 +2777,42 @@ QUEST_20151016_002784
+QUEST_20151016_002788	Ferret
 QUEST_20151016_002789	{memo X}
 QUEST_20151016_002790	{memo X}
 QUEST_20151016_002791	{memo X}
-QUEST_20151016_002792	
-QUEST_20151016_002793	
-QUEST_20151016_002794	
-QUEST_20151016_002795	
-QUEST_20151016_002796	
+QUEST_20151016_002792	Freed girl has ran away somewhere with urgency.
+QUEST_20151016_002793	Girl looked at my way as if she wanted me to follow her.
+QUEST_20151016_002794	Urbonas
+QUEST_20151016_002795	From succession, We've spent a long time preparing to greet the Revelator.
+QUEST_20151016_002796	Gathering Shining Moss.
 QUEST_20151016_002797	{memo X}
 QUEST_20151016_002798	{memo X}
 QUEST_20151022_002799	Collapsed Residential Quarters
 QUEST_20151022_002800	Collapsed Residential Quarters (Recommended Lv: 160){nl}Only you and your party members can enter together.{nl}Do you want to enter?
-QUEST_20151022_002801	
-QUEST_20151022_002802	
-QUEST_20151022_002803	
-QUEST_20151022_002804	
-QUEST_20151022_002805	
-QUEST_20151022_002806	
-QUEST_20151022_002807	
-QUEST_20151022_002808	
-QUEST_20151022_002809	
-QUEST_20151022_002810	
-QUEST_20151022_002811	
-QUEST_20151022_002812	
-QUEST_20151022_002813	
-QUEST_20151022_002814	
-QUEST_20151022_002815	
-QUEST_20151022_002816	
-QUEST_20151022_002817	
-QUEST_20151022_002818	
-QUEST_20151022_002819	
-QUEST_20151022_002820	
-QUEST_20151022_002821	
-QUEST_20151022_002822	
-QUEST_20151022_002823	
+QUEST_20151022_002801	Are you the one who rescued us?{nl}Then we shall lend you our strength.
+QUEST_20151022_002802	Thank you for saving us from this awful work.{nl}It might be small, but please accept my strength.
+QUEST_20151022_002803	How hard it has been.. If it weren't for you, we would've spent eternity working.{nl}Thank you. Let me share my strength.
+QUEST_20151022_002804	Take this strength.{nl}If this awful tower can be destroyed, I will do anything.
+QUEST_20151022_002805	By any chance, are you going to leave after receiving our strength?{nl}Since it's request by the General Carved Owl, I will lend my strength.
+QUEST_20151022_002806	My strength to be weak since it hasn't been long since I recovered my conscious.{nl}However, I hope that even this small amount of strength my help.
+QUEST_20151022_002807	Letter
+QUEST_20151022_002808	Even after running away from the demons that has invaded the castle, it'll be hard to survive. If anyone get hold of this letter, please pass it on to Yurus.. Eras
+QUEST_20151022_002809	Pillar is engraved with 1.
+QUEST_20151022_002810	Pillar is engraved with 2.
+QUEST_20151022_002811	Pillar is engraved with 3.
+QUEST_20151022_002812	Something is written faintly. A symbol.. A shard..?
+QUEST_20151022_002813	Thus, I decided to interfere.{nl}Even if it meant that I would be targetted by the demons.
+QUEST_20151022_002814	Area near us is the Kedoran Merchant Alliance's excavation site.{nl}Be cautious to cause no disturbance to the excavation.
+QUEST_20151022_002815	Detector is indicating it's close in the upper right corner.
```

Comment:
The detector

it's close = that it is close?

(applies to the next 8 lines as well)